packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -0,0 +1,123 @@
+import { Button, Box, Grid } from 'grommet'
+import { FormNext, FormPrevious } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled, { withTheme } from 'styled-components'
+import { useTranslation } from 'react-i18next'
+
+import locationValidator from '../../../helpers/locationValidator'
+
+const StyledButton = styled(Button)`
+  & > div {
+    flex-direction: column;
+  }
+`
+
+const backgrounds = { dark: 'dark-3', light: 'neutral-6' }
+
+const FlipbookControls = ({
+  currentFrame = 0,
+  locations = [],
+  onFrameChange = () => true,
+  theme
+}) => {
+  const { t } = useTranslation('components')
+
+  const handleFrameChange = frameIndex => {
+    onFrameChange(frameIndex)
+  }
+
+  const handlePrevious = () => {
+    if (currentFrame > 0) {
+      handleFrameChange(currentFrame - 1)
+    }
+  }
+
+  const handleNext = () => {
+    if (currentFrame < locations.length) {
+      handleFrameChange(currentFrame + 1)
+    }
+  }
+
+  return (
+    <Box background={backgrounds}>
+      <Grid columns={['120px', 'flex']} pad='10px'>
+        {/** Play/Pause & Speed go here */}
+        <Box />
+
+        {/** Image Thumbnails */}
+        <Box direction='row' justify='between'>
+          <StyledButton
+            disabled={currentFrame === 0}
+            icon={<FormPrevious />}
+            label={t(
+              'SubjectViewer.MultiFrameViewer.FrameCarousel.previousFrameLabel'
+            )}
+            onClick={() => handlePrevious()}
+            style={{
+              border: 'none',
+              padding: 0
+            }}
+          />
+          <Box
+            aria-label='Image thumbnails'
+            direction='row'
+            gap='5px'
+            pad='5px'
+          >
+            {locations?.length &&
+              locations.map((location, index) => {
+                const mimeType = Object.keys(location)[0]
+                const url = location[mimeType]
+                const activeFrame = currentFrame === index
+
+                return (
+                  <Button
+                    key={`${url}-${index}`}
+                    aria-label={`${t('SubjectViewer.MultiFrameViewer.FrameCarousel.thumbnailAltText')} ${index + 1}`}
+                    aria-selected={activeFrame}
```

Comment:
All the ARIA states are strings, so the value here should be `'true'` or `'false'`. I don't think ARIA has any boolean attributes.

```suggestion
                    aria-selected={activeFrame ? 'true' : 'false'}
```

Reply:
Also, [`aria-selected`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected) isn't valid for buttons, so the semantics here will have to change. See the linked MDN page for roles that are allowed to use this state.

Reply:
[`aria-pressed`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed) is valid for buttons. If you don't want to change the button roles, then I'd suggest changing this to:
```suggestion
                    aria-pressed={activeFrame ? 'true' : 'false'}
```

Then keep the list as a list of buttons for the time being.